aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -217,6 +217,30 @@ def to_dict(self, origin: Optional[str]) -> Dict[str, str]:
             headers["Access-Control-Allow-Credentials"] = "true"
         return headers
 
+    @staticmethod
+    def extract_origin_header(resolver_headers: Dict):
+        """
+        Extracts the 'origin' or 'Origin' header from the provided resolver headers.
+
+        The 'origin' or 'Origin' header can be either a single header or a multi-header.
+
+        Args:
+            resolver_headers (Dict): A dictionary containing the headers.
+
+        Returns:
+            Union[str, List[str], None]: The value(s) of the 'origin' or 'Origin' header.
+                If the header is a single header, a string is returned.
+                If the header is a multi-header, a list of strings is returned.
+                If the header is not present, None is returned.
+        """
+        resolved_header = resolver_headers.get("origin") or resolver_headers.get("Origin")
```

Comment:
can we use our internal utility function to get header and use case_sensitive=False so it lowers it?

That should avoid any other permutations of the header we can't predict

Reply:
Good catch @heitorlessa.